app/resources/create.html.template

```diff
@@ -572,11 +571,9 @@
 
         </div>
       </div>
-      </div>
 
-      <div class="column end"></div>
       <div class="end-multi-columns"></div>
```

Comment:
Well, end-multi-columns in the end of each set of fields are required per documented here:
https://github.com/google/personfinder/blob/master/app/resources/css-default.template#L243
So only delete the last one I pointed, and keep the remaining ones.